### ISSUE TITLE
尝试微调qwen3-embedding-0.6b没有进度条

### ISSUE THREAD

**Issue by Godlikemandyy** [modelscope/ms-swift#5053]
今天尝试微调一下qwen3-embedding-0.6b，没有进度条和日志输出

<img width="1080" height="1187" alt="Image" src="https://github.com/user-attachments/assets/0b8593a8-2779-4abb-8ddd-2666c3d21a28" />
为什么没有进度条呢，是需要设置某个参数还是没有呢

**Comment by Godlikemandyy** [comment]
> 今天尝试微调一下qwen3-embedding-0.6b，没有进度条和日志输出
> 
> <img alt="Image" width="1080" height="1187" src="https://private-user-images.githubusercontent.com/36065226/469068218-0b8593a8-2779-4abb-8ddd-2666c3d21a28.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMxNzE5NTcsIm5iZiI6MTc1MzE3MTY1NywicGF0aCI6Ii8zNjA2NTIyNi80NjkwNjgyMTgtMGI4NTkzYTgtMjc3OS00YWJiLThkZGQtMjY2NmMzZDIxYTI4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA3MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNzIyVDA4MDczN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFkYWM1Zjk2NjBjYjM3YzkwZDRhNTY4MzBlZTE1ZTI4ZDg2YzdkMmFiZjc1NTkxNTA4NjE0YzQ1MjBiZTAzYTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.rSOq-etpm3dmox9vpmQmrhdNo9Z2CLhGqlPdNjAm5lY"> 为什么没有进度条呢，是需要设置某个参数还是没有呢。

参数如下：
`CUDA_VISIBLE_DEVICES=0 \
swift sft \
--model /root/autodl-tmp/swift_scripts/Qwen3-Embedding-0.6B \
--task_type embedding \
--model_type qwen3_emb \
--train_type full \
--dataset test.jsonl \
--split_dataset_ratio 0.05 \
--eval_strategy steps \
--output_dir output \
--save_steps 50 \
--eval_steps 50 \
--num_train_epochs 1 \
--per_device_train_batch_size 4 \
--per_device_eval_batch_size 4 \
--gradient_accumulation_steps 4 \
--learning_rate 6e-6 \
--loss_type infonce \
--label_names labels \
--dataloader_drop_last true \
--deepspeed zero2`
